### PR TITLE
Bgs 2.3

### PR DESCRIPTION
**Pull request by bwuphan**

### CODE REVIEW

File: _data/services.yml [hunk 1]
```diff
@@ -1,33 +1,36 @@
...
+  link   : project-management
+  point  : Need someone to handle the daily grind? We’ll make sure everyone knows what’s going on.
+- name   : Staff Augmentation
+  image  : staff_augmentation_icon.png
+  link   : staff-augmentation
+  point  : If you need to scale your current team, or need experts in a specific area, we’re ready to go.
+- name   : Support & Maintenance
+  image  : support_and_maintenance_icon.png
+  link   : support-and-maintenance
+  point  : If we made it for you, we stand behind it. From changes to new features , we’re here for you.
```
Comment (aajungroth) [inline _data/services.yml]: Remove extra space after:
From changes to new features

File: capabilities/backend-development.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+tags: [Back End Development]
+
+title: Back-End Development
+
+subtitle: "It gets no fanfare, but it does all the hard work invisibly."
+
+text: '<p>How it works. You store your data somewhere - likely the cloud - and then give
+users a way to interact with that data through APIs. That’s the basic gist of back-end.</p>
+<p>APIs are considered back-end development. The code that runs on the back end of the
+server. The front end runs on a browser. The API makes the call to the back-end. It
```
Comment (aajungroth) [inline capabilities/backend-development.html]: Remove this sentence:
The API makes the call to the back-end.

File: capabilities/data-warehousing.html [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+“People don’t really realize how useful data is.”</p>
+<p class="font-size-md">She was right. We all now understand how collecting and reporting
+data is critical for the success of a business. You grab it, shape it, visualize it, and
+plan your future with it!</p>
+<p class="font-size-md">It’s a pretty broad umbrella term but let’s use this example.
+Anytime you’re storing a large amount of data, especially historical data, you want to be
+able to look back. For our client AmericasMart, their exhibitors sell their products, but
+they also sold products last year, or two or three years ago. We have all this historic
+data warehoused, so they can generate reports on who was there, when, and in which booth
+We can build you a warehouse and combine many different types of data (link to Sys Int)
```
Comment (aajungroth) [inline capabilities/data-warehousing.html]: Add the link to:
(link to Sys Int)

File: capabilities/devops.html [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+efficiency and speed of deployment. See how we do it."
+
+tags: [DevOps]
+
+title: DevOps
+
+subtitle: "It’s developers and operations working together to safely (but quickly) get your code out to your servers."
+
+text: '<p>We do DevOps for all our clients. </p>
+<p>Let’s say you have a database, or two, or 300. There’s your security component, (link
```
Comment (aajungroth) [inline capabilities/devops.html]: Add a link to:
(link to security page)

File: capabilities/devops.html [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+to security page), the authentication and authorization system. There’s a session server
```
Comment (aajungroth) [inline capabilities/devops.html]: Add a link to:
(link to security page)

File: capabilities/devops.html [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+which ensures that happens.</p>
+<p>Or maybe you’re just focused on one piece. Say one of five servers need to be updated.
+How do you do that in a way so the other four don’t fail?</p>
+<p>Operations sets up the infrastructure. We write your code. Then we test the code. If
+any of those tests fails, we go back and fix it. After that, the code moves out to
+servers.</p>
+<p>Developers do certain things to make the operations team feel good about what they’re
+doing. The operations team does certain things to make the developers feel confident in
+their work.</p>
+<p>The intersection of Agile development and people is DevOps.</p>'
```
Comment (aajungroth) [inline capabilities/devops.html:23]: This article doesn't explain the topic accurately enough. We need to discuss this further.

File: capabilities/project-management.html [hunk 7]
```diff
@@ -0,0 +1,17 @@
+---
+layout: capability
+
+asset-name: project_management
+
+description: "Offload project management to us and we’ll watch over the day-to-day execution  so you can focus on bigger stuff."
```
Comment (aajungroth) [inline capabilities/project-management.html]: Remove a space after:
Offload project management to us and we’ll watch over the day-to-day execution

File: capabilities/security.html [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+your business. See what we can offer. This is one of our strengths!"
+
+tags: [Security]
+
+title: Security
+
+subtitle: "Absolutely the most important element of a software."
+
+text: '<p>We have a lot of experience specifically with Single Sign On.</p>
+<p>Google, Facebook, GitHub, and Twitter do this – we do too. and many more . It’s the
```
Comment (aajungroth) [inline capabilities/security.html]: This sentence has punctuation and spacing errors:
Google, Facebook, GitHub, and Twitter do this – we do too. and many more .

Suggested fix:
Google, Facebook, GitHub, Twitter and many more do this; we do too.

File: capabilities/system-integrations.html [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+they need access to the data. We have set up numerous APIs to work off their existing
+database. Easy for us, and the customer only sees the end result – a nightmare avoided. </p>
+<p>Another example would be if somebody came to us and said, “I have this application and
+it’s already been built; it’s been running and now I want to take payments on it.” We have
+experience taking payments for all sorts of things, and then integrating with a bunch of
+different merchants. We have had clients whose business takes payments, but their system
+isn’t PCI compliant – that compliance can be expensive. Instead, we set them up with a
+payment merchant (like authorize.Net, or PayPros) that will take the credit card details.
+Our client only stores these things called tokens in their system, which can only be used
+by that system.</p>
```
Comment (aajungroth) [inline capabilities/system-integrations.html]: This sentence needs to be discussed. I think we need to be cautious around statements about tokens as they are  prone to security vulnerabilities. 
Maybe more details about how the tokens are used to interface with the payment system could help.

File: _data/services.yml (hunk 1 above)
Comment (johnscottBG) [reply _data/services.yml]: Yes

File: capabilities/backend-development.html (hunk 2 above)
Comment (johnscottBG) [reply capabilities/backend-development.html]: Yes

File: capabilities/data-warehousing.html (hunk 3 above)
Comment (johnscottBG) [reply capabilities/data-warehousing.html]: Yes please

File: capabilities/devops.html (hunk 5 above)
Comment (johnscottBG) [reply capabilities/devops.html]: Yes please

File: capabilities/devops.html (hunk 6 above)
Comment (johnscottBG) [reply capabilities/devops.html:23]: Sounds good

File: capabilities/project-management.html (hunk 7 above)
Comment (johnscottBG) [reply capabilities/project-management.html]: Yes

File: capabilities/security.html (hunk 8 above)
Comment (johnscottBG) [reply capabilities/security.html]: Yes

File: capabilities/system-integrations.html (hunk 9 above)
Comment (johnscottBG) [reply capabilities/system-integrations.html]: Yes

File: _data/services.yml (hunk 1 above)
Comment (bwuphan) [reply _data/services.yml]: Fixed.

File: capabilities/backend-development.html (hunk 2 above)
Comment (bwuphan) [reply capabilities/backend-development.html]: Removed sentence plus removed 'The front end runs on a browser.'

File: capabilities/data-warehousing.html (hunk 3 above)
Comment (bwuphan) [reply capabilities/data-warehousing.html]: Fixed.

File: capabilities/devops.html (hunk 5 above)
Comment (bwuphan) [reply capabilities/devops.html]: Fixed.

File: capabilities/project-management.html (hunk 7 above)
Comment (bwuphan) [reply capabilities/project-management.html]: Fixed.

File: capabilities/security.html (hunk 8 above)
Comment (bwuphan) [reply capabilities/security.html]: Fixed.

File: capabilities/devops.html [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+deployment. See how we do it."
+
+tags: [DevOps]
+
+title: DevOps
+
+subtitle: "It’s developers and operations working together to safely (but quickly) get your code out to your servers."
+
+text: '<p>We do DevOps for all our clients. </p>
+<p>Traditionally, developers write code, operations deploys it. But what happens in a siloed if something goes wrong?
```
Comment (aajungroth) [inline capabilities/devops.html]: Add the word, environment, after:
But what happens in a siloed

Comment (johnscottBG) [reply capabilities/devops.html]: Yes, thanks!

File: capabilities/software-development.html [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+subtitle: Crafting solutions to automate your process
+
+text: 'Here at BG Software, we understand that there are many business needs that can be solved with Software.
+Through meeting with our clients and discussing their current workflow and processes, we help to design/craft
+the best possible solution, often
+<a class="color-teal hover-color-dark-blue" href="/case-studies/americasmart">automating</a> processes that were
+previously done manually and/or on paper. The software development process captures all the steps in between. Because
+the business is involved in requirements gathering and available for help in the design phase, the end product is unique
+and suits them. Software development doesn’t have to be a big project or application. It can be a small custom
+calculator on a website, adding new fields toan already existing submittal form, or implementing a single sign-on for
```
Comment (aajungroth) [inline capabilities/software-development.html]: Add a space after:
adding new fields to

File: capabilities/staff-augmentation.html [hunk 12]
```diff
@@ -0,0 +1,25 @@
...
+description: "Our software development team can also be yours, if you need some help. Learn
+more about our staff augmentation services."
+
+tags: [Staff Augmentation]
+
+title: Staff Augmentation
+
+subtitle: "...so you can concentrate on what you’re good at."
+
+text: "<p>TThere are a lot of companies who need custom software, who have a budget for custom software, but don’t
```
Comment (aajungroth) [inline capabilities/staff-augmentation.html]: Remove the extra T in:
TThere

File: capabilities/software-development.html (hunk 11 above)
Comment (johnscottBG) [reply capabilities/software-development.html]: Yes

File: capabilities/staff-augmentation.html (hunk 12 above)
Comment (johnscottBG) [reply capabilities/staff-augmentation.html]: Yep

Comment (bwuphan) [reply capabilities/staff-augmentation.html]: My fault. Fixed.

File: capabilities/software-development.html (hunk 11 above)
Comment (bwuphan) [reply capabilities/software-development.html]: My fault. Fixed.

File: capabilities/devops.html (hunk 6 above)
Comment (johnscottBG) [reply capabilities/devops.html:23]: Fixed

File: capabilities/system-integrations.html (hunk 9 above)
Comment (johnscottBG) [reply capabilities/system-integrations.html]: Fixed.

**Pull request merged**